### PR TITLE
Add privacy policy, italicize FIRST

### PR DESCRIPTION
**Pull request by cmacfarl**
Fixes https://github.com/FIRST-Tech-Challenge/ftcdocs/issues/140

Also brings the copyright into conformance with FIRST policy that FIRST always be italicized.  And makes the format consistent with the copyright notice on ftc-scoring which drops the text 'Copyright' as that's redundant with the &#169; symbol.  And yes, I realize ftc-events has the redundancy at present.  Note that the year was pulled out of conf.py because there was no way to italicize FIRST without also italicizing the year as long as the year was part of the copyright property in the conf file.

### CODE REVIEW

**Comment by cmacfarl** [comment]
image

It would be nice to be able to collapse that extra space above the copyright.

**Review by texasdiaz — CHANGES_REQUESTED** [review]
Can we do the same for the Terms and Conditions?

https://www.firstinspires.org/sites/default/files/uploads/first-terms-conditions.pdf

**Comment by cmacfarl** [comment]
> Can we do the same for the Terms and Conditions?
> 
> https://www.firstinspires.org/sites/default/files/uploads/first-terms-conditions.pdf

Do you want Report an Incident also?

https://www.pavesuite.com/FIRST/PublicPortal/HomePage

File: docs/source/_templates/footer.html [hunk 1]
```diff
@@ -35,6 +35,12 @@
   <div>
     <a href="https://www.firstinspires.org/about/privacy-policy">Privacy Policy</a>
   </div>
+  <div>
+    <a href="/tos/tos.html">Terms of Service</a>
+  </div>
+  <div>
+    <a href="https://www.pavesuite.com/FIRST/PublicPortal/HomePage">Report an Incident</a>
```
Comment (cmacfarl) [inline docs/source/_templates/footer.html:42]: Didn't you say on Thursday you didn't want this on ftc-docs?  Did you change your mind?

Comment (texasdiaz) [reply docs/source/_templates/footer.html:42]: I totally did say that. However, Chris Rake overrode me in the Teams chat we had after our meeting when he said:
image

Therefore, we should add it.

**Comment by cmacfarl** [comment]
https://github.com/FIRST-Tech-Challenge/ftcdocs/pull/143/commits/208f658b1b857c92562f96f3765c64434e79170c cleans up the footer a bit.  With that commit Season Presenting Sponsor is vertically aligned with the FIRST copyright.

image

**Review by texasdiaz — APPROVED** [review]

**Pull request merged**